.github/styles/config/vocabularies/Papermoon/reject.txt

```diff
@@ -3,10 +3,7 @@ her
 hers
 him
 his
-let's
```

Comment:
Why are these terms removed from the reject list?

Reply:
Because these words will flag under a rule in the Google ruleset. This is eliminating one of the reasons the same words were flagging more than once for each occurrence.